thirdparty-config/classes/Race-ExRace.conf

```diff
@@ -1,10 +1,35 @@
-#Name must be unique betweem races, classes, and skilltrees
-Name: ExRace
-MenuIcon: "minecraft:gold_ore"
+# More indepth information can be found on the Ore wiki for NT-RPG
+#
+#"Name" must a string and must be unique betweem races, classes, and skilltrees
+"Name": "ExRace"
+#"ClassType" must be a string and must be defined in the settings.conf file
+"ClassType": "Race"
+#ItemType must be a string and follow the "modid:itemname" nomenclature.
+ItemType: "minecraft:gold_ore"
+#Description displayed in game; must be a string.
 Description: "Describe your race here"
-#the Name of the class files which you want your race to beable to use, must match the Name: variable within the <classname>.conf file.
-AllowedClasses: [
-  ExClass
+AttributePointsPerLevel: 0
+SkillPointsPerLevel: 1
+#see the wiki for more information about leveling.
+Leveling: {
+    __class__: cz.neumimto.rpg.players.leveling.Exponential
```

Comment:
cz.neumimto.rpg.players.leveling.Exponential should be "cz.neumimto.rpg.players.leveling.Exponential"

Reply:
Fixed